src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitJobServlet.groovy

```diff
@@ -147,6 +147,28 @@ class GrabbitJobServlet extends SlingAllMethodsServlet {
         response.writer.write(new JsonBuilder(jobExecutionIds).toString())
     }
 
+    @Override
+    protected void doPost(SlingHttpServletRequest request, SlingHttpServletResponse response)  {
+        String jobExecutionId = request.getParameter("jobId") ?: ""
+        if(!jobExecutionId.isLong()) {
+            log.warn "Parameter ${jobExecutionId} 'jobId' is invalid"
```

Comment:
Sure. Take a look at https://github.com/TWCable/grabbit/blob/master/src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitJobServlet.groovy#L56

You can use "all" as a special meta-parameter to query all known jobs, e.g:
```
GET /grabbit/job/all
```

So what I'm saying here is - if someone tries to
```
DELETE /grabbit/job/all
``` 
we should not tell them that the jobID 'all' is invalid, because it's not. We should tell them deleting 'all' is unsupported or something